api/src/main/java/org/cardanofoundation/rosetta/api/block/model/domain/PoolRetirement.java

```diff
@@ -6,7 +6,7 @@
 import lombok.NoArgsConstructor;
 
 @Data
-@Builder // TODO saa: remove this and refactor tests
```

Comment:
Same question as Dimitri already had at the previous change.

Reply:
Answered